build.sbt

```diff
@@ -1,8 +1,8 @@
-name := "AkkaStreamsCommandPipe"
+name := "shell-command-flow"
 
 organization := "io.dronekit"
 
-version := "1.0"
+version := "1.1-M2"
```

Comment:
Does M2 need to be in the package version? Why not just 1.0 since this is a different package now?